bin/pocs_shell

```diff
@@ -310,7 +313,7 @@ Hardware names: {}   (or all for all hardware)'''.format(
 
         self.pocs = None
 
-    def do_polar_alignment_test(self, *arg):
+    def do_polar_alignment_utils_test(self, *arg):
```

Comment:
I think this change was a mistake, i.e. a global search and replace of polar_alignment to polar_alignment_utils that went too far.

Reply:
Oops, yeah. Good case for why this file needs testing. :) I'll get it changed.